packages/server/medplum.config.json

```diff
@@ -38,5 +38,6 @@
     "removeOnComplete": { "count": 1 }
   },
   "shutdownTimeoutMilliseconds": 30000,
-  "chainedSearchWithReferenceTables": true
+  "chainedSearchWithReferenceTables": true,
+  "logRequests": true
```

Comment:
Adding `"logRequests": true` to the default config. This does make logs slightly noisier but we already have this enabled in prod, and having this off made us miss a bug in the logging middleware before.

Having this on will make sure we catch any bugs that would happen in our new e2e tests

Reply:
@mattlong  added a test specifically for that case, so i think we can leave this off: https://github.com/medplum/medplum/pull/5756

Or i suppose we could use Jest's silent mode: https://jestjs.io/docs/cli#--silent

Reply:
Alright we can remove it. I guess that test should at least help us to detect most kinds of catastrophic failures